components/SubmissionForm.vue

```diff
@@ -26,25 +26,25 @@
             data-testid="submit-input-googlemaps"
             type="text"
             required
-            class="mb-5 px-3 py-3.5 w-[350px] h-[50px] bg-white rounded-lg border border-zinc-400
-          text-neutral-600 text-sm font-normal font-['Noto Sans JP']"
+            class="mb-5 px-3 py-3.5 w-96 h-12 bg-white rounded-lg border border-primary-text-muted
```

Comment:
instead of `bg-white` try one of the theme colors.

Reply:
We don't have a bg color in our theme that is white, but we do have `primary-text-inverted` the only thing is maybe it's confusing because of the name? I can make a new bg theme color as well?

The bg-white was based on the tailwind config file.

Reply:
https://github.com/ourjapanlife/findadoc-web/blob/ermish/backup-loginform/components/LoginForm.vue

Take a look at what i did for these login form backgrounds that are similar. I used bg-secondary
You can add a new one, but make sure it fits with the theme. When playing with colors, I recommend try adding `theme-neon` to the top div in the app vue file to see if it looks nice on darker themes (I plan to release a dark theme eventually so it's good to test against it)

Reply:
the secondary-bg is a coral color, like this:

![image](https://github.com/ourjapanlife/findadoc-web/assets/114712265/b302837c-bd95-4c7f-9734-c0778fbb4f45)

Reply:
Up to you! You can use any of them or add a new one 👍 but make sure to add it across the themes if you add one 🙃

Reply:
I think by-primary-bg is another color option. You can always just check the ones available in our css file with the theme colors